config/default.yml

```diff
@@ -174,6 +174,18 @@ Bundler/GemComment:
   IgnoredGems: []
   OnlyFor: []
 
+Bundler/GemVersionDeclaration:
+  Description: 'Requires or prohibits gem version declarations.'
+  Enabled: false
+  VersionAdded: '1.13.0'
+  EnforcedStyle: 'required'
+  SupportedStyles:
+    - 'required'
+    - 'prohibited'
+  Include:
+    - '**/Gemfile'
```

Comment:
I saw that other `Bundler` cops also include `*.gemfile` and `gems.rb`. I've personally never worked with this format of gem file, so I was unsure what exactly to test.

Reply:
The format is exactly the same, it's just a different filename.